src/pages/Login/Signup.styles.scss

```diff
@@ -0,0 +1,364 @@
+@import url('https://fonts.googleapis.com/css?family=Biryani&display=swap');
+
+body {
+    background: #050516;
+    overflow: hidden;
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center; /*centers items on the line (the x-axis by default)*/
+    align-items: center; /*centers items on the cross-axis (y by default)*/
+
+    .signup-component {
+        background: #0d4e67;
+        border-radius: 4px;
+        width: 500px;
```

Comment:
I suggest avoiding using fixed width and height because it's most of the time mess up with responsiveness.